control-plane/rest/src/versions/v0.rs

```diff
@@ -12,6 +12,16 @@ use std::{
     string::ToString,
 };
 use strum_macros::{self, Display};
+use types::v0::message_bus::mbus::{
```

Comment:
this should be pub use to export the types which are part of the rest client interface. The user of the rest client interface should only require to use the rest_client library - in theory :)

Reply:
Done.